pkg/blockfactory/blockissuer.go

```diff
@@ -426,6 +390,32 @@ func (i *BlockIssuer) AttachBlock(ctx context.Context, iotaBlock *iotago.Protoco
 	return modelBlock.ID(), nil
 }
 
+func (i *BlockIssuer) setDefaultBlockParams(blockParams *BlockParams) error {
```

Comment:
#279 might change this again but still great to get rid of this duplicate code

Reply:
:+1: